tests/framework/installer/install_helper.go

```diff
@@ -387,18 +409,21 @@ func (h *InstallHelper) CleanupCluster(clusterName string) {
 	logger.Infof("Uninstalling All Rook Clusters - %s", clusterName)
 	_, err := h.k8shelper.DeleteResource([]string{"-n", clusterName, "cluster", clusterName})
 	if err != nil {
-		logger.Errorf("Rook Cluster  %s cannot be deleted,err -> %v", clusterName, err)
+		logger.Warningf("Rook Cluster  %s cannot be deleted,err -> %v", clusterName, err)
```

Comment:
Looks like there is an extra space between `Cluster` and `%s`.

Reply:
indeed!